_data/sites.json

```diff
@@ -9328,15 +9328,25 @@
 
     {
         "name": "League of Legends",
-        "url": "https://support-leagueoflegends.riotgames.com/hc/en-us/articles/202647784-Account-Deletion-FAQ",
+        "url": "https://support-leagueoflegends.riotgames.com/hc/en-us/articles/360050328454",
         "difficulty": "hard",
-        "notes": "<a href=\"https://support.riotgames.com/anonymous_requests/new\">Send Riot Games a ticket</a> with the following information: account name, summoner name, server, email address used for account registration, registration location (City, Country), last IP used to play League of Legends, Internet Service Provider used during registration, and payment methods used to purchase RP. (if applicable)",
-        "notes_tr": "Şu bilgilerle bir destek talebi gönderin: hesap adı, sihirdar adı, sunucu, hesap kaydı için kullanılan e-posta adresi, kayıt konumu (Şehir, Ülke), League of Legends oynamak için kullanılan son IP, kayıt sırasında kullanılan İnternet Servis Sağlayıcısı ve kullanılan ödeme yöntemleri RP satın almak için. (eğer varsa)",
+        "notes": "Go to the linked page, scroll down, click on \"LOG IN\" and confirm your deletion.",
```

Comment:
Seems to be just clicking a button now, wouldn't it make LoL easy, then?

The same applies to all other entries you updated/added

Reply:
@Kefaku ?

Reply:
I'm currently on vacation, I'll be back next wednesday, can't do anything till then @tupaschoal

Reply:
No problem, enjoy your time off!

Reply:
just updated, @tupaschoal